packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/splitter.tests.js

```diff
@@ -39,6 +39,80 @@ const moduleConfig = {
     }
 };
 
+QUnit.module('Resizing', moduleConfig, () => {
+    function assertLayout(items, expectedLayout, assert) {
+        items.toArray().forEach((item, index) => {
+            assert.strictEqual(item.style.flexGrow, expectedLayout[index]);
+        });
+    }
+
+    ['horizontal', 'vertical'].forEach(orientation => {
+        QUnit.test(`items should be evenly distributed by default with ${orientation} orientation`, function(assert) {
+            this.reinit({
+                orientation,
+                dataSource: [{ }, { }]
+            });
+
+            const items = this.$element.find(`.${SPLITTER_ITEM_CLASS}`);
+
+            assertLayout(items, ['50', '50'], assert);
+        });
+    });
+
+    [[50, ['75', '25'], 'horizontal'], [-50, ['25', '75'], 'horizontal'], [-100, ['0', '100'], 'horizontal'],
```

Comment:
It will more readable if you will use the following notation
```
[{
   resizeDistance: ...,  expectedSize: ..., orientation : ...
}].forEach({ resizeDistance, expectedSize, orientation })
```